### ISSUE TITLE
Problems in windows build 

### ISSUE THREAD

**Issue by trsh** [djc/askama#96]
```
error[E0405]: cannot find trait `TraitFromapplications2fapi5cbase2ehtml` in this scope
  --> manu\src\applications\api\views\default.rs:10:10
   |
10 | #[derive(Template)]
   |          ^^^^^^^^ did you mean `TraitFromapplications2fapi2fbase2ehtml`?
```

**Comment by djc** [comment]
Hey, thanks for the report and sorry about the breakage! What version is this? Can you give me a more complete minimal reproducer? Have you verified that it works on non-Windows, or are you just testing on Windows?

**Comment by trsh** [comment]
Yes it works on Ubuntu with same code and version.
Im installing from cargo: `askama = "0.6"`
Min example:

```
use askama::Template;

#[derive(Template)]
#[template(path = "applications/api/base.html")]
struct BaseTemplate;

#[derive(Template)]
#[template(path = "applications/api/default.html")]
struct DefaultTemplate<'a> {
    name: &'a str,
    _parent: BaseTemplate,
}
```
Windows is 10 x64, rust using MSVC

**Comment by djc** [comment]
I'm afraid you might be the first person trying it on Windows, then! Or possibly the first person on Windows using inheritance. I'm setting up an AppVeyor build right now so I can do some interactive debugging.

**Comment by djc** [comment]
BTW, if you could try Askama from the master branch, that would also be great. It in fact seems to pass testing on AppVeyor:

https://ci.appveyor.com/project/djc/askama

**Comment by djc** [comment]
So this seems to have something to do with how the paths for parent templates are constructed. In particular, the trait defined for the parent template is named `TraitFromapplications2fapi5cbase2ehtml` (for `applications/api/base.html`) whereas the child template is looking for a trait named `TraitFromapplications2fapi5cbase2ehtml` (for `applications/api\base.html`). I think this would happen if you used the relative path -- just `base.html` -- from inside `default.html`. If you instead use `applications/api/base.html` it will probably work correctly.

**Comment by trsh** [comment]
SO what's the verdict? It no problem in master, on stable use it like this?

**Comment by djc** [comment]
I'm pretty close (hopefully today) to a new release that won't have this problem, so if you can use the workaround for now, that would be great, yes.

**Comment by djc** [comment]
I've just released 0.7.0, which should not have this problem.

**Comment by trsh** [comment]
tnx
